### ISSUE TITLE
Upgrade neighbourhood shapes for Cleveland

### ISSUE THREAD

**Issue by nvkelso** [whosonfirst-data/whosonfirst-data#404]
### tl;dr

Status:
- [ ] Propose **sources**
- [ ] Establish **concordances** between WOF and source
- [ ] **Upgrade / add** WOF shapes
- [ ] **Add new or missing** features to WOF
- [ ] **Deprecate / supersede** WOF features
- [ ] **Draft review**
- [ ] **Draft revisions** based on review comments
- [ ] **Final review**
- [ ] **Final revisions** based on review comments
- [ ] **Import** into Who's On First

### Details

Sourcing:
- Permissive license?
- Add new src to Who's On First

Upgrade / add:
- `macrohood`
- `neighbourhood`
- `microhood`

Add new or missing features to WOF
- `macrohood`
- `neighbourhood`
- `microhood`

Deprecate / supersede:
- `macrohood`
- `neighbourhood`
- `microhood`

**Comment by nvkelso** [comment]
Possible sources:

- Wikipedia: 
  - https://en.wikipedia.org/wiki/Cleveland#Neighborhoods 
- Good narrative: 
  - http://www.streetdirectory.com/travel_guide/209273/north_america_destinations/cleveland_ohio_neighborhoods.html 
- official Statistical Planning Areas (they are in fact used)
 - http://planning.city.cleveland.oh.us/gis/cpc/basemap.jsp?command=larger
- In PDF form: http://planning.city.cleveland.oh.us/maps/jpg/All_SPAs_2014.jpg 
- See long list of local maps of various kinds at: 
  - http://planning.city.cleveland.oh.us/maps/maplist.php?pageNum_rs1=0&totalRows_rs1=155 
- http://www.rustbelt.com/portfolio/Downtown_2007.pdf

**Comment by skorasaurus** [comment]
ref: 
https://github.com/whosonfirst-data/whosonfirst-data/issues/189

I've been in touch with @stepps00 on it in the past couple weeks and gave me the starter kit for it. 

(Some of this includes my response to him in an email): 

The city of Cleveland does not have any defined neighbourhood boundaries (only statistical planning areas that are mostly exclusive to the City Planning department). 

I actually created some of my own neighborhood boundaries a couple years ago (I hadn't heard of alpha or betashapes) - https://skorasaurus.wordpress.com/cleveland-neighborhood-map/ that I synthesized from a variety of sources, personal knowledge and asking residents and CDCs. 

Since that last published release of the work 

@stepps00 mentioned that he'd prefer some that is completely contiguous but I wouldn't feel extremely confident on defining boundaries for some parts of the city. If you're comfortable and A-OK with the best that I can do, that's good for me and I'd fill in the gaps as best that I can with existing records and 

Lastly, what's your timeframe on it to have it completed?
